ktor-jetty/src/org/jetbrains/ktor/jetty/CommandLine.kt

```diff
@@ -17,7 +15,9 @@ fun main(args: Array<String>) {
     }
 
     val jar = map["-jar"]?.let { File(it).toURI().toURL() }
-    val combinedConfig = ConfigFactory.load()
+    val argConfig = ConfigFactory.parseMap(map.filterKeys { it.startsWith("-P:") }.mapKeys { it.key.removePrefix("-P:") }, "Command-line options")
+    val combinedConfig = argConfig.withFallback(ConfigFactory.load())
+
```

Comment:
Ok, I see. Then we probably should drop it from this PR, implement reusable command-line configuration facility as we need this in Netty command line and any other host we might have.